plugins/generator-1.18.1/forge-1.18.1/templates/livingentity/livingentity_renderer.java.ftl

```diff
@@ -83,7 +83,7 @@ public class ${name}Renderer extends <#if humanoid>Humanoid</#if>MobRenderer<${n
 		</#if>
 
 		<#if data.mobModelGlowTexture?has_content>
-		this.addLayer(new EyesLayer(this) {
+		this.addLayer(new EyesLayer<${name}Entity, ${model}>(this) {

```

Comment:
During my tests, I noticed it wasn't necessary. Is there a particular reason to have it?

Reply:
Generics should be used when possible, it is Javanic thing to do to keep types in check